src/templates/_macros/feed_item.html

```diff
@@ -225,7 +225,7 @@ <h1 class="feed-item-name">{{ coll.name|translate(coll)|safe }}</h1>
                  src="{{ feed_item.background_image_landing or feed_item.background_image }}"
                  alt="" draggable="false" itemprop="contentURL">
           {% endif %}
-          <div class="feed-landing-header-inner">
+          <div class="feed-landing-header-inner {{ 'has-desc' if feed_item.description }}">
```

Comment:
make this a property of the .feed-landing-wrapper where all the other stuff is and prefix the class name